bloco_4/dia_1/maior-de-tres.js

```diff
@@ -0,0 +1,9 @@
+let a = 5, b = 25, c = 75;
+
+if (a>b && a>c) {
+    console.log("a é maior");
+} else if (b>a && b>c){
+    console.log("b é maior");
+} else {
+    console.log("c é maior");
```

Comment:
Nesse caso, se o `c` for igual a `a` ou `b`, o retorno será de que c é o maior dos três! Sugiro a mesma mudança do outro exercício!